### PR TITLE
New "make a great listing" page

### PR DESCRIPTION
**Pull request by awfuchs**
To complement the extension best practices, a new page for the CWS listing page itself.

[Page preview](https://deploy-preview-1124--developer-chrome-com.netlify.app/docs/webstore/best_listing)

Lightly adapted from a draft by @lsauser .

### CODE REVIEW

File: site/en/docs/webstore/best_listing/index.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
   Web Store.
 ---
 
-The Chrome Web Store's focus on surfacing quality items to users encompases the entire user experience,
-including the store listing page. Build trust with users and make a strong first impression with a
-compelling and accurate store listing page.  Learn how to create a quality store listing page that
-clearly communicates what your item will offer from the description, images, and other metadata
-provided. You can build and maintain your store listing page through the developer dashboard.
+The Chrome Web Store works hard to surface quality items for users to discover.  Making sure yours
+is a "quality item" can help ensure its more prominence in the store, potentially increasing your
```
Comment (lsauser) [inline site/en/docs/webstore/best_listing/index.md:11]: remove move?

File: site/en/docs/webstore/best_listing/index.md [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+installed base.
```
Comment (lsauser) [inline site/en/docs/webstore/best_listing/index.md:12]: install base, or user base?

File: site/en/docs/webstore/best_listing/index.md (hunk 1 above)
Comment (awfuchs) [reply site/en/docs/webstore/best_listing/index.md:11]: Oops. Done.

File: site/en/docs/webstore/best_listing/index.md (hunk 2 above)
Comment (awfuchs) [reply site/en/docs/webstore/best_listing/index.md:12]: Yeah, I'll go with user base. Done.

**Review by lsauser — APPROVED** [review]
lgtm

**Pull request merged**